src/index.ts

```diff
@@ -1,4 +1,8 @@
+import { eddsa } from 'elliptic';
+
 export * from './memory';
 export * from './backend';
 export * from './instance';
 export * from './environment';
+
+global.eddsa = () => global._eddsa || (global._eddsa = new eddsa('ed25519'));
```

Comment:
We want to store this globally (and lazy-initialise it) because constructing it each time is slow - on my year-old macbook, about 500ms.